src/index.html

```diff
@@ -11,12 +11,97 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo of company MOYO"
+          class="moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item name-1">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item name-2">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item name-3">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item name-4">
+            <a
+              href="#laptopsandcomputers"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item name-5">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item name-6">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item name-7">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item name-8">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```